package.json

```diff
@@ -100,6 +101,11 @@
                     "maximum": 1000,
                     "markdownDescription": "%AWS.configuration.description.s3.maxItemsPerPage%"
                 },
+                "aws.s3.defaultRegion": {
```

Comment:
Why do we need to expose this to customers?

Reply:
This is for S3 calls where we can't infer a region. e.g. what happens if a user has listBucket permissions in us-west-2 but is blocked from making it in us-east-1?

Reply:
> e.g. what happens if a user has listBucket permissions in us-west-2 but is blocked from making it in us-east-1?

How will the user know that they need to fiddle with this setting? Can we iterate the regions and figure out which one didn't fail, then store it (without exposing as a setting).

Reply:
I think VS Code lets you link to the setting via command? We should have some sort of a notification to alert them. Will take a look.

Reply:
What about displaying an error when the call with the default 'us-east-1' fails?

Reply:
Just added an error message displayed when region was invalid with a link to the settings, also changed the description so that it makes more sense what the setting is for, then added a dropdown menu with possible regions